### PR TITLE
Apply typographical fixes to zed-animal.json

### PR DESCRIPTION
**Pull request by ToxiClay**
#### Summary

SUMMARY: None

#### Purpose of change

This PR addresses typographical issues in `zed-animal.json`.

#### Describe the solution

I went over the file and edited it.

#### Describe alternatives you've considered

None.

#### Testing

Linting is clean.

### CODE REVIEW

File: data/json/monsters/zed-animal.json [hunk 1]
```diff
@@ -342,7 +342,7 @@
     "id": "mon_zombie_horse",
     "type": "MONSTER",
     "name": { "str": "zombie horse" },
-    "description": "From the looks of this zombie horse's ghastly features, with its protruding ribs, whitish skull, and empty eyes, the exposed part of the internal organs shows a seemingly lifeless color, and the black body fluid drips slowly.  The new strength makes the horse no longer dependent on its muscles, but it can still pursue the enemy quickly.",
+    "description": "From the looks of this zombie horse's ghastly features--its protruding ribs, off-white skull, and decayed, tattered muscles--it doesn't seem like it should put up much of a fight, but its mutant strength gives it power. Black, reeking ichor drips from its wounds.",
```
Comment (BrettDong) [inline data/json/monsters/zed-animal.json:345]:
```suggestion
    "description": "From the looks of this zombie horse's ghastly features--its protruding ribs, off-white skull, and decayed, tattered muscles--it doesn't seem like it should put up much of a fight, but its mutant strength gives it power.  Black, reeking ichor drips from its wounds.",
```

File: data/json/monsters/zed-animal.json [hunk 2]
```diff
@@ -470,7 +470,7 @@
     "type": "MONSTER",
     "name": { "str": "woodland wight" },
     "looks_like": "mon_deer",
-    "description": "Pale, stumbling white-tailed deer with blackened eyes and drooling slime down its neck.  Its legs twist and bend in strange, haphazard directions, but with unnatural strength and speed.  Preys on coyotes, wolves, and giant spider mutants.",
+    "description": "A pale, stumbling, white-tailed deer, rivulets of slime running down its neck. Its limbs twist and bend in unnatural, haphazard directions, and yet it moves with uncanny strength and speed. Thanks to the Cataclysm, it has now turned the tables, and what was once prey, now itself preys on coyotes, wolves, and giant mutated spiders.",
```
Comment (BrettDong) [inline data/json/monsters/zed-animal.json:473]:
```suggestion
    "description": "A pale, stumbling, white-tailed deer, rivulets of slime running down its neck.  Its limbs twist and bend in unnatural, haphazard directions, and yet it moves with uncanny strength and speed.  Thanks to the Cataclysm, it has now turned the tables, and what was once prey, now itself preys on coyotes, wolves, and giant mutated spiders.",
```

**Review by BrettDong — CHANGES_REQUESTED** [review]
Needs two spaces between sentences.

File: data/json/monsters/zed-animal.json [hunk 3]
```diff
@@ -230,7 +230,7 @@
     "id": "mon_zombeaver",
     "type": "MONSTER",
     "name": { "str": "zombeaver" },
-    "description": "This wall-eyed beast has a large wound on its side, and ribs with some flesh on them are seen through it.  The most prominent feature of this abomination is large incisors in its ominously open mouth, strange black goo is oozing out of it.  Seems like it isn't interested in trees anymore and is looking for some warm flesh to feed.",
+    "description": "This wall-eyed beast has a large wound on its side, and inside, its gore-stained ribs are clearly visible.  Its most prominent features are the fearsome incisors and thick, black goo in its mouth that it seems proud to display.  It doesn't look interested in chewing on trees any longer.",
```
Comment (actual-nh) [inline data/json/monsters/zed-animal.json:233]:
```suggestion
    "description": "This wall-eyed beast has a large wound on its side, and inside, its gore-stained ribs are clearly visible.  Its most prominent features are the fearsome incisors and thick, black goo in its mouth that it seems proud to display.  It doesn't look interested in chewing on trees anymore.",
```

File: data/json/monsters/zed-animal.json [hunk 4]
```diff
@@ -548,7 +548,7 @@
     "type": "MONSTER",
     "name": { "str": "Zeindeer" },
     "looks_like": "mon_zeer",
-    "description": " A once majestic Caribou, this undead creature barely resembles the reindeer of tradition and seems to only be interested in human flesh.",
+    "description": " A once-majestic caribou, this undead creature barely resembles the reindeer of tradition and seems to only be interested in human flesh.",
```
Comment (BrettDong) [inline data/json/monsters/zed-animal.json:551]: Is the initial space here intentional, or typo?

Comment (ToxiClay) [reply data/json/monsters/zed-animal.json:551]: It was like that when I first looked at the file; it's not a change I introduced, so I can't answer that. I missed it in my initial pass.

**Review by BrettDong — APPROVED** [review]
<del>Changes are technically okay now: it passes JSON validation, and game loads successfully.</del>

File: data/json/monsters/zed-animal.json [hunk 5]
```diff
@@ -470,7 +470,7 @@
     "type": "MONSTER",
     "name": { "str": "woodland wight" },
     "looks_like": "mon_deer",
-    "description": "Pale, stumbling white-tailed deer with blackened eyes and drooling slime down its neck.  Its legs twist and bend in strange, haphazard directions, but with unnatural strength and speed.  Preys on coyotes, wolves, and giant spider mutants.",
+    "description": "A pale, stumbling, white-tailed deer, rivulets of slime running down its neck. Its limbs twist and bend in unnatural, haphazard directions, and yet it moves with uncanny strength and speed.  Thanks to the Cataclysm, it has now turned the tables, and what was once prey, now itself preys on coyotes, wolves, and giant mutated spiders.",
```
Comment (actual-nh) [inline data/json/monsters/zed-animal.json:473]:
```suggestion
    "description": "A pale, stumbling, white-tailed deer, rivulets of slime running down its neck.  Its limbs twist and bend in unnatural, haphazard directions, and yet it moves with uncanny strength and speed.  Thanks to the Cataclysm, it has now turned the tables; what was once prey, now itself preys on coyotes, wolves, and giant mutated spiders.",
```
Space is definitely needed. Latter part is stylistic.

Comment (BrettDong) [reply data/json/monsters/zed-animal.json:473]: Oh I thought this should have been caught by JSON validation.

**Review by BrettDong — APPROVED** [review]
CI passes and game loads locally.

**Pull request merged**